jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JacksonModule.java

```diff
@@ -98,6 +98,10 @@ public void applyToConfigBuilder(SchemaGeneratorConfigBuilder builder) {
             generalConfigPart.withPropertySorter(new JsonPropertySorter(true));
         }
 
+        if (this.options.contains(JacksonOption.RESPECT_JSONPROPERTY_REQUIRED)) {
+            fieldConfigPart.withRequiredCheck( this::getRequiredCheckBasedOnJsonPropertyAnnotation );
```

Comment:
Let's be consistent with existing similar code:
```suggestion
            fieldConfigPart.withRequiredCheck(this::getRequiredCheckBasedOnJsonPropertyAnnotation);
```